docs/source/llms/llm-evaluate/index.rst

```diff
@@ -234,6 +234,7 @@ metrics:
 * :py:func:`mlflow.metrics.genai.answer_relevance`: Use this metric when you want to evaluate how relevant the model generated output is to the input (context is ignored). High scores mean that your model outputs are about the same subject as the input, while low scores mean that outputs may be non-topical.
 * :py:func:`mlflow.metrics.genai.relevance`: Use this metric when you want to evaluate how relevant the model generated output is with respect to both the input and the context. High scores mean that the model has understood the context and correct extracted relevant information from the context, while low score mean that output has completely ignored the question and the context and could be hallucinating.
 * :py:func:`mlflow.metrics.genai.faithfulness`: Use this metric when you want to evaluate how faithful the model generated output is based on the context provided. High scores mean that the outputs contain information that is in line with the context, while low scores mean that outputs may disagree with the context (input is ignored).
+* :py:func:`mlflow.metrics.bleu`: Use this metric when you want to evaluate the quality of machine-generated text, particularly for translation tasks. BLEU (Bilingual Evaluation Understudy) measures the similarity between the model's output and one or more reference texts. Scores range from 0 to 1, with 1 indicating a perfect match. Note that while BLEU is widely used, it has limitations such as not accounting for meaning or grammatical correctness, so it's often used in conjunction with other metrics for a more comprehensive evaluation.
```

Comment:
here we only mention the metrics that are using LLM_as_judge to evaluate the model. So not sure if we want to add this documentation here. 
cc @BenWilson2 for the docs side changes

Reply:
I think it's a reasonable spot. The section above it is for default evaluators, which this won't have a direct mapping :)

Reply:
Good point! I removed the entry as pointless